templates/blog/post_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Post's List:</h1>
+  <ul>
+    {% for post in post_list%}
```

Comment:
There is a missing space between 'post_list' and '%}'. This might cause a syntax error in the template rendering.